src/SignalR/clients/java/signalr/src/test/java/com/microsoft/signalr/PersonPojo.java

```diff
@@ -0,0 +1,40 @@
+package com.microsoft.signalr;
+
+public class PersonPojo<T> implements Comparable<PersonPojo<T>> {
```

Comment:
Why comparable?

Reply:
Some descriptions of POJO claim they can't implement interfaces - wanted to test that msgpack-jackson could still serialize an object that did so